docs/EmbeddedAppIOS.md

```diff
@@ -161,6 +161,21 @@ In root directory, we need to start React Native development server.
 
 This command will start up a React Native development server within our CocoaPods dependency to build our bundled script. The `--root` option indicates the root of your React Native apps – this will be our `ReactComponents` directory containing the single `index.ios.js` file. This running server will package up the `index.ios.bundle` file accessible via `http://localhost:8081/index.ios.bundle`.
 
+## Update App Transport Security
+
+On iOS 9 and above the app won't be a able to connect over http to localhost unless specifically told so. See this thread for alternatives and instructions: http://stackoverflow.com/questions/31254725/transport-security-has-blocked-a-cleartext-http.
```

Comment:
Just randomly jumping in here, but I think RN-based docs _and_ an RN-based tool would help. The tool could generate a patch like below for people's app’s `Info.plist` file:

```diff
diff --git a/Resources/Info.plist b/Resources/Info.plist
index 715a99a..2c38673 100644
--- a/Resources/Info.plist
+++ b/Resources/Info.plist
@@ -24,6 +24,17 @@
 	<string>140</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
+			<dict>
+				<key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
 	<key>UIAppFonts</key>
 	<array>
 		<string>PT_Sans-Web-Regular.ttf</string>
```

Similarly, in a `react-native init`-generated project, it should not be [completely disabling App Transport Security](https://github.com/facebook/react-native/blob/f8f5cb1fcf893397ccb8172921e8dfaad8ff6150/local-cli/generator-ios/templates/app/Info.plist#L41-L46), but registering an exception for `localhost` as above.